esp/bindings/bindutil.cpp

```diff
@@ -724,3 +724,25 @@ IEspHttpException* createEspHttpException(int code, const char *_msg, const char
 {
     return new CEspHttpException(code, _msg, _httpstatus);
 }
+
+int executeCommand(const char* command, StringBuffer& response)
+{
+    char buffer[128];
+#ifdef _WINDOWS
```

Comment:
I don't think you need to specialize for windows.  platform.h maps popen to _popen for windows.

Reply:
So I think you can just use popen below.